designs/waralee/src/waistband.mjs

```diff
@@ -11,182 +11,127 @@ function waraleeWaistband(
-  points.zigzagBottomL = points.zigzagBottom.shift(180, waistBand / 8)
-  points.zigzagBottomB = points.zigzagBottom.shift(270, waistBand / 4)
-
-  paths.ZZtop = new Path()
-    .move(points.zigzagTopR)
-    .line(points.zigzagTopT)
-    .line(points.zigzagTopB)
-    .line(points.zigzagTopL)
-    .attr('class', 'dotted')
-  paths.ZZbottom = new Path()
-    .move(points.zigzagBottomR)
-    .line(points.zigzagBottomT)
-    .line(points.zigzagBottomB)
-    .line(points.zigzagBottomL)
-    .attr('class', 'dotted')
-  paths.right = new Path()
-    .move(points.zigzagBottomR)
-    .line(points.br)
-    .line(points.mr)
-    .line(points.tr)
-    .line(points.zigzagTopR)
-  paths.left = new Path()
-    .move(points.zigzagBottomL)
-    .line(points.bl)
-    .line(points.ml)
-    .line(points.tl)
-    .line(points.zigzagTopL)
+  if (!expand) {
+    // Expand is on, do not draw the part but flag this to the user
+    store.flag.note({
+      msg: `waralee:cut` + type,
+      replace: {
+        width: units(waistBand * 2),
+        length: units(waistBandLength),
+      },
+      suggest: {
+        text: 'flag:show',
+        icon: 'expand',
+        update: {
+          settings: ['expand', 1],
+        },
+      },
+    })
+    // Also hint about expand
+    store.flag.preset('expand')
+
+    return part.hide()
+  }
+
+  store.cutlist.addCut({ cut: 1, from: 'fabric' })
+
+  points.tl = new Point(0, 0)
+  points.bl = new Point(0, waistBandLength)
+  points.tm = new Point(waistBand, 0)
+  points.bm = new Point(waistBand, waistBandLength)
+  points.tr = new Point(waistBand * 2, 0)
+  points.br = new Point(waistBand * 2, waistBandLength)
 
   paths.seam = new Path()
     .move(points.br)
-    .line(points.mr)
     .line(points.tr)
     .line(points.tl)
-    .line(points.ml)
     .line(points.bl)
     .line(points.br)
     .close()
-    .hide()
-
-  // Complete?
-  if (complete) {
-    if (sa) paths.sa = paths.seam.offset(sa).attr('class', 'fabric sa')
-
-    points.title = points.tl.shiftFractionTowards(points.br, 0.2).shift(270, 20)
-    macro('title', {
-      nr: partNr,
-      at: points.title,
-      title: type,
-      prefix: 'front',
-      scale: 0.4,
-    })
 
-    points.logo = points.tl.shiftFractionTowards(points.br, 0.8)
-    snippets.logo = new Snippet('logo', points.logo).attr('data-scale', 0.3)
-  }
-  // Paperless?
-  if (paperless) {
-    macro('vd', {
-      id: 1,
-      from: points.tr,
-      to: points.br,
-      x: points.br.x + sa + 10,
-    })
-  }
+  paths.fold = new Path()
+    .move(points.tm)
+    .line(points.bm)
+    .attr('class', 'fabric dotted')
+    .setText('fold')
+
+  if (sa) paths.sa = paths.seam.offset(sa).attr('class', 'fabric sa')
+
+  store.cutlist.addCut({ cut: 1, from: 'fabric' })
+
+  points.title = new Point(waistBand, waistBandLength * 0.2)
+  macro('title', {
+    nr: partNr,
+    at: points.title,
+    title: type,
+    prefix: 'front',
```

Comment:
The title macro no longer uses `prefix`. If you want to prefix your title, please use `front${type}` as `title` value.